x/mint/types/keys.go

```diff
@@ -1,8 +1,11 @@
 package types
 
-// MintKey is the key to use for the keeper store.
+// MintKey is the key to use for the Minter in the keeper store.
 var MintKey = []byte{0x00}
 
+// GenesisTimeKey is the key to use for GenesisTime in the keeper store.
+var GenesisTimeKey = []byte{0x01}
```

Comment:
[Non blocking]
I think something like this would be better so that if you're dumping the tree, you don't have to check what `0x01` means
```suggestion
var GenesisTimeKey = []byte{"MintGenesisTime"}
```
or similar

Reply:
Similar for the mint module

Reply:
> Similar for the mint module

Clarifying question, do you mean the `MintKey` on line 4?

Reply:
Yes

Reply:
Resolved via https://github.com/celestiaorg/celestia-app/pull/1851/commits/37d67309f4416186c4aedae41bf2262b0a59b453